src/tool_helpers.c

```diff
@@ -78,6 +78,8 @@ const char *param2text(int res)
     return "error encountered when reading a file";
   case PARAM_EXPAND_ERROR:
     return "variable expansion failure";
+  case PARAM_BLANK_STRING:
+    return "blank argument where content were expected";
```

Comment:
s/were/was/

Reply:
Thanks, I think I will even go with **is**.